packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -210,12 +212,34 @@ export const OutboundShipmentDetailViewComponent: FC = () => {
       </Modal>
 
       <AppBarContentPortal
-        sx={{ display: 'flex', flex: 1, justifyContent: 'center' }}
+        sx={{
+          display: 'flex',
+          flex: 1,
+          justifyContent: 'center',
+          paddingLeft: '25px',
+        }}
       >
-        <TabList value={currentTab} onChange={onChangeTab}>
-          <Tab value="general" label={t('label.general')} />
-          <Tab value="transport" label={t('label.transport')} />
-        </TabList>
+        <Box display="flex" flex={1}>
+          <InputWithLabelRow
+            Input={
+              <CustomerSearchInput
+                value={draft.name}
+                onChange={name => {
+                  draft.update?.('name', name);
+                }}
+              />
+            }
+          />
+        </Box>
+
+        <Box display="flex" flex={1} justifyContent="center">
+          <TabList value={currentTab} onChange={onChangeTab}>
+            <Tab value="general" label={t('label.general')} />
+            <Tab value="transport" label={t('label.transport')} />
+          </TabList>
+        </Box>
+
+        <Box display="flex" flex={1} justifyContent="flex-end" />
```

Comment:
Is it time to put a save and cancel button on the bottom of this component 🤔

Reply:
Ha yes I think so!